plottable.d.ts

```diff
@@ -1251,7 +1258,7 @@ declare module Plottable {
         class Line extends Drawer {
             constructor(dataset: Dataset);
             protected _applyDefaultAttributes(selection: d3.Selection<any>): void;
-            selectionForIndex(index: number): d3.Selection<void>;
```

Comment:
Yey...
I guess I can force cast it. If we look above at the drawer, the `selectionForIndex` actually returns a `d3.Selection<any>`. It's weird that this one diverged.